2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -485,3 +485,57 @@ A causa dei rallentamenti incontrati, alcuni ruoli hanno visto un monte ore effe
 Il ruolo dell'amministratore invece, incaricato di gestire il reparto tecnologico implementando gli automatismi necessari, a causa di difficoltà incontrate nell'implementazione (errori, testing, verifica), ha visto un aumento delle ore rispetto a quanto preventivato.
 === Burndown-chart
 [da inserire]
+
+
+== Secondo sprint dal 06-11-2023 al 13-11-2023
+
+=== Obiettivi raggiunti
+- riconfigurazione del repository;
+- stesura della sezione _introduzione_ del documento delle Norme di Progetto;
+- stesura della sezione _processi di supporto_ del documento delle Norme di Progetto;
+- perfezionamento delle automazioni per la compilazione dei documenti;
+- perfezionamento delle automazioni per il versionamento dei documenti;
+- contatto con l'azienda Proponente per comunicare l'esito della candidatura;
+- meeting con l'azienda Proponente per riflettere sull'Analisi dei Requisiti e sulle ulteriori tecnologie da usare;
+- inizio dell'individuazione e della stesura degli use case.
+=== Obiettivi mancati
+Gli obiettivi sono stati tutti raggiunti, considerando anche che molti erano obiettivi mancati dello sprint precedente.
+=== Analisi problematiche insorte
+- Il team ha notato la mancanza di una struttura comune nei verbali, che porta a documenti senza una precisa convenzione e rallentamenti in fase di redazione;\
+- Sono emerse difficoltà nelle modalità di utilizzo del repository comune, nonostante sia stato dedicato del tempo per la formazione;
+- Scarsa reattività in fase di review.
+=== Risoluzioni attuate
+- Per risolvere la problematica di una mancanza di struttura nei verbali, si sono fissate delle convenzioni da seguire nel documento di Norme di Progetto;
+- Per formare meglio il team sulle modalità di utilizzo del repository, è stato dedicato tempo in più per fornire tutorial video e testuali in supporto del team;
+- Per ovviare alla scarsa reattività in fase di review, il team prende la responsabilità di visionare spesso la casella mail personale per
```

Comment:
qua e in altri posti sei andato a capo a caso, so che su typst non lo fa vedere poi, ma preferirei venisse sistemato tanto è easy

Reply:
sistemato